scripts/build_sailfish.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
+#run on the Sailfish OS sdk virtual machine. Check that rpmbuild directory exists. Remember to export VERSION_ID
```

Comment:
Please update comment, that this is to be used on Docker sailfish build environment. It would not correctly build on traditional VirtualBox SDK because of the changed paths. It's up to you if you want to keep my hands-off-marker below.